src/test/java/edu/ksu/canvas/attendance/repository/AttendanceRepositoryImplUTest.java

```diff
@@ -42,8 +42,9 @@ public void saveInBatches_tollerateNullAttendances() {
     public void deleteAttendanceByCourseAndDayOfClass_NullDateOfClass() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)